src/KingSystem/Ecosystem/ecoLevelSensor.cpp

```diff
@@ -20,4 +21,55 @@ void LevelSensor::init(sead::Heap* heap) {
     mRootIter = new (heap) al::ByamlIter(res->getRawData());
 }
 
+void LevelSensor::calculatePoints()
+{
+    if (mDefaultPoints >= 0) {
+        mPoints = mDefaultPoints;
+    }
```

Comment:
considering returning early here to reduce unnecessary indentation

Reply:
mismatch, code at the end